src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goBack(value = 1) {
+      if (value && value > 0) {
```

Comment:
value в будь-якому випадку існує
```suggestion
      if (value > 0) {
```